commcare_connect/templates/tailwind/base.html

```diff
@@ -1,39 +1,30 @@
-{% load static i18n %}
-
 <!DOCTYPE html>
-{% get_current_language as LANGUAGE_CODE %}
-<html lang="{{ LANGUAGE_CODE }}">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>{% block title %}CommCare Connect{% endblock title %}</title>
-    {% block css %}
-    <link rel="stylesheet" href="{% static 'bundles/css/tailwind.css' %}">
-    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
-    {% endblock %}
-    {% block javascript %}
-    <script src="{% static 'bundles/js/tailwind-bundle.js' %}" defer></script>
-    <script src="https://unpkg.com/htmx.org"></script>
-
-    {% endblock javascript %}
-</head>
-<body class="bg-gray-100 flex overflow-x-hidden font-work-sans">
-
-    <!-- Sidebar -->
-    {% include 'tailwind/sidebar.html' %}
-
-    <!-- Main Content -->
-    <div class="flex-1">
-        <!-- Header -->
-        {% include 'tailwind/header.html' %}
-        <!-- Content Area (Two Columns) -->
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CommCareConnect</title>
+    {% load static %}
+    <script defer src="https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js"></script>
```

Comment:
We already have alpinejs and htmx available via webpack, you just need to include this file https://github.com/dimagi/commcare-connect/blob/main/commcare_connect/static/js/vendors.js